src/ir/module-utils.cpp

```diff
@@ -222,7 +231,6 @@ void copyModule(const Module& in, Module& out) {
   out.customSections = in.customSections;
   out.debugInfoFileNames = in.debugInfoFileNames;
   out.features = in.features;
-  out.typeNames = in.typeNames;
```

Comment:
Why did this move? The description of the function it was moved to is that it copies toplevel module items like Functions and Globals, and not metadata.

Reply:
So, you think I should rather put the code that copy type names in `wasm-merge.cc`?

Reply:
Sorry, I don't see why my question, which was about the deletion of this line, is answered by something about `wasm-merge`? What is the context I am missing?

From the standpoint of this file by itself (not thinking about `wasm-merge` at all, this change seems surprising as I said - it looks like this code moved up to `copyModuleItems` which I don't understand the motivation for.

Reply:
`wasm-merge` currently does not copy type names from the input modules to the merged module. I added some code for that in `copyModuleItems`. But then this line became unnecessary.

Reply:
I see, so wasm-merge uses `copyModuleItems` and not `copyModule`. The difference between those two decreases with this change, and the meaning of the difference becomes less clear to me. Perhaps we don't need two functions here at all, and there could just be a single function with maybe a set of options as to whether to copy exports, the start method, etc. (the things only in `copyModule`). Anyhow, for this PR I think the change you made is good, but please add a TODO on `copyModule` to remind us later, maybe something like `TODO: merge this with copyModuleItems, and add options for copying exports and other things that are currently different between them, if we still need those differences`.